README.md

```diff
@@ -1,2 +1,73 @@
-# raincatcher
-Feedhenry Raincatcher Core Repository
+# RainCatcher
+Feedhenry RainCatcher Core Repository
+
+## About RainCatcher
+
+## Developing
+
+RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
+
+Check your Node.JS version running `node -v`
+
+### Setup
+
+```bash
+git clone https://github.com/feedhenry-raincatcher/raincatcher.git
```

Comment:
Should this be git clone `https://github.com/feedhenry-raincatcher/raincatcher-core.git`